frameworks/kafka/src/main/dist/svc.yml

```diff
@@ -88,6 +141,37 @@ pods:
           export KAFKA_OPTS="-Dsun.security.krb5.debug=true $KAFKA_OPTS"
         {{/TASKCFG_ALL_SECURITY_KERBEROS_DEBUG}}
       {{/TASKCFG_ALL_SECURITY_KERBEROS_ENABLED}}
+      {{#TASKCFG_ALL_SECURE_JMX_ENABLED}}
+          export SECURE_JMX_KEY_STORE_KEY=`cat $MESOS_SANDBOX/jmx/key_store_pass`
+          export KEY_STORE_PATH=$MESOS_SANDBOX/jmx/key_store
+          if [ -f "$MESOS_SANDBOX/jmx/trust_store" ]; then
+              # load user provided trust store
+              export TRUST_STORE_PATH=$MESOS_SANDBOX/jmx/trust_store
+              export SECURE_JMX_TRUST_STORE_KEY=`cat $MESOS_SANDBOX/jmx/trust_store_pass`
+          else
+              # load user provided keystore certs to default truststore
+              export TRUST_STORE_PATH=$JAVA_HOME/lib/security/cacerts
+              export SECURE_JMX_TRUST_STORE_KEY="changeit"
+              self_signed_aliases=$(${JAVA_HOME}/bin/keytool -list -v -keystore $KEY_STORE_PATH -storepass $SECURE_JMX_KEY_STORE_KEY | grep 'Alias name:' | awk '{print $3}')
+              for cert_alias in $self_signed_aliases ; do
+                  ${JAVA_HOME}/bin/keytool -importkeystore -srckeystore $KEY_STORE_PATH -destkeystore $TRUST_STORE_PATH -srcstorepass $SECURE_JMX_KEY_STORE_KEY -deststorepass $SECURE_JMX_TRUST_STORE_KEY -srcalias $cert_alias -destalias $cert_alias ;
```

Comment:
Looking at the docs, you probably want a `-noprompt` here to prevent hanging on a password mismatch?
Also, please drop the trailing semicolon, it's not needed I think.

Reply:
good catch!